ExtraEnemyCustomization/CustomAbilities/DrainStamina/DrainStaminaManager.cs

```diff
@@ -0,0 +1,32 @@
+﻿using EECustom.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace EECustom.CustomAbilities.DrainStamina
+{
+    [CallConstructorOnLoad]
+    public static class DrainStaminaManager
+    {
+        internal static DrainStaminaSync Sync { get; private set; } = new();
```

Comment:
Why is this internal?

Reply:
API purpose, if I open Sync as public, random assembly might directly use Sync event instead of given methods